manifests/service.pp

```diff
@@ -1,27 +1,25 @@
-# Define: registry::service
-#
+# @summary Defined resource type that manages service entries
+# @note
 #   This defined resource type manages service entries in the Microsoft service
 #   control framework by managing the appropriate registry keys and values.
 #
 #   This is an alternative approach to using INSTSRV.EXE [1].
```

Comment:
Lines 6-9 are not included as part of the note tag.

Reply:
Is this still an issue that needs to be fixed?